go.mod

```diff
@@ -103,6 +103,7 @@ require (
 	github.com/valyala/tcplisten v0.0.0-20161114210144-ceec8f93295a
 	github.com/xiang90/probing v0.0.0-20190116061207-43a291ad63a2 // indirect
 	go.etcd.io/bbolt v1.3.3 // indirect
+	go.opencensus.io v0.21.0
```

Comment:
there is no reason to include this as direct dependency @nitisht can you remove this change?